src/services/PlantingsService.ts

```diff
@@ -21,7 +21,7 @@ const listPlantings = async (
       'delivery.plantings.id',
       'delivery.plantings.plantingSite.id',
       'delivery.plantings.plantingSubzone.id',
-      'delivery.plantings.plantingSubzone.totalPlants(raw)',
+      'delivery.plantings.plantingSubzone.totalPlants',
```

Comment:
using string value so it comes pre-formatted/localized.. we won't be using raw numbers for math

Reply:
going to switch back to (raw) since the string values don't sort well in the table.. will use the FormatterNumber in the renderer instead